feature-libs/pdf-invoices/components/pdf-invoices-components.module.ts

```diff
@@ -45,6 +45,9 @@ import { InvoicesListComponent } from './invoices-list/invoices-list.component';
           guards: [AuthGuard],
         },
       },
+      features: {
+        useUpdatedSortMappingForInvoices: false,
```

Comment:
I recommend starting with the name of the library, e.g. `pdfInvoices` and if possible describe semantically the change: e.g. `sortByInvoiceDate`.
What do you think about: `pdfInvoicesSortByInvoiceDate`?

Reply:
yes, it is better. have changed the name